search-app/src/cmr/search/services/query_execution.clj

```diff
@@ -146,7 +146,9 @@
         elastic-results (idx/execute-query context processed-query)
         query-results (rc/elastic-results->query-results context query elastic-results)
         query-results (if (or (tc/echo-system-token? context) (:skip-acls? query))
-                        query-results
+                       ;;TODO JYNA temporarily forcing it to go past system token
+                       ; query-results
+                        (filter-query-results-with-acls context query-results)
```

Comment:
I think you need to let the system token work.  Please remove the comment on line 150 and remove lines 149 and 151.

Reply:
No worries on this. I was going to revert it back before the official review. It is here temporarily so I can force it to bypass echo token for my own local testing. My local env is strangely forcing everything under system-token even when I don't pass it a token, so I changed it in code temporarily. It is safe to ignore this PR until it is out of DRAFT mode.